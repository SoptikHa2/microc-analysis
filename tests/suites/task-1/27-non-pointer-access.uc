 main() {
   var r;
   r = 1;
   (*r).x = 41;
   return r.x;
 }
