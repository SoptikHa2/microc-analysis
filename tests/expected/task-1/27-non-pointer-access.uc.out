---
test: run suites/task-1/27-non-pointer-access.uc
---
Analysis found some problems. Aborting run.
InvalidRecordField "In main, \"suites/task-1/27-non-pointer-access.uc\" (line 4, column 9): Accessing unknown record field: x"
InvalidRecordField "In main, \"suites/task-1/27-non-pointer-access.uc\" (line 5, column 13): Accessing unknown record field: x"

exit: 1
